static/locales/en-US.yaml

```diff
@@ -270,6 +270,10 @@ History:
   Empty Search Message: There are no videos in your history that match your search
   Search bar placeholder: "Search in History"
   Case Sensitive Search: Case Sensitive Search
+  Sort By:
+    Sort By: Sort By
```

Comment:
We really need to deduplicate this string, as this pull request will add the 6th `Sort By` string/text, I'll do a separate pull request for that though.

Reply:
do I leave it like this for now then? or hard code it in the interim (since this string doesn't have sort anyways?) or maybe reference another string for now?